core/src/main/java/org/elasticsearch/common/util/concurrent/QueueResizingEsThreadPoolExecutor.java

```diff
@@ -43,17 +44,20 @@
  */
 public final class QueueResizingEsThreadPoolExecutor extends EsThreadPoolExecutor {
 
+    public static double EWMA_ALPHA = 0.3;
```

Comment:
I'm curious about why you chose this. If you don't have a good reason for that number then maybe leave a comment about how this is just a guess.